.github/ISSUE_TEMPLATE/meeting-deliverables.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "🚀 Deliverables Review Meeting"
+about: Conduct a deliverables review meeting.
+labels: "type: team-sync, :label: deliverables-meeting"
+title: "Deliverables Review Meeting: <YYYY-MM-DD>"
+---
+
+# 2i2c Deliverables Review meeting
+
+This is a **30 minute** recurring meeting every two weeks on **Monday at 8:30AM Pacific time** (here's [**a timezone converter**](https://arewemeetingyet.com/Los%20Angeles/2000-01-01/08:30/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=), ignore the date!).
+
+Our goal is to synchronize the team on the most important things to work on, and to divide work between one another so we know what to work on next.
+
+- **Video conference link:** https://bit.ly/zoom-holdgraf
+- **Calendar for future meetings:** [this Google Calendar](https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com&ctz=America%2FLos_Angeles)
+
+## Meeting agenda
+
+_Follow the agenda below, checking boxes as we complete each step._
+
+### Activity Backlog
+_(5m) Quick review of [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to let team members signal-boost if needed._
+
+- [ ] Quick team report on what they're up to right now, where they could use help.
```

Comment:
I think there is an overlapping between this point and the team sync issue... Maybe the meeting should happen every other **Tuesday** so everyone has time to look at other's reports and look into the board by themselves?